AcceleratorSampleApp/app/src/main/java/com/tokbox/android/accelerator/sample/MainActivity.java

```diff
@@ -651,6 +673,11 @@ public void onVideoQualityWarningLifted(OTWrapper otWrapper, String remoteId) th
                     Log.i(LOG_TAG, "The quality has improved");
                 }
 
+                @Override
+                public void onAudioLevelUpdated(float v) throws ListenerException {
+
+                }
+
```

Comment:
Any comment or log that explains this method and necessity of  float value  could be helpful.